.cargo/config.toml

```diff
@@ -21,4 +21,7 @@ ci-license = "license-template --template .license_template"
 #
 # - RUSTSEC-2021-0065: https://rustsec.org/advisories/RUSTSEC-2021-0065
 #   - anymap is unmaintained 🤷‍♀️
-ci-audit = "audit --file Cargo.lock --deny warnings --ignore RUSTSEC-2021-0065"
+# - RUSTSEC-2023-0052: https://rustsec.org/advisories/RUSTSEC-2023-0052
+# - RUSTSEC-2023-0053: https://rustsec.org/advisories/RUSTSEC-2023-0053
+#   - TODO: waiting for fix in dependency
+ci-audit = "audit --file Cargo.lock --deny warnings --ignore RUSTSEC-2021-0065 --ignore RUSTSEC-2023-0052  --ignore RUSTSEC-2023-0053"
```

Comment:
I don't think we should add 53 :(
It was already gone if we updated the deps

Reply:
ugh fiiine